react/TextNote/styles.styl

```diff
@@ -0,0 +1,8 @@
+@require '../../stylus/generic/typography'
+@require '../../stylus/utilities/text'
+
+.g-text-note
+    @extend $text-note
+
+.u-ellipsis
+    @extend $ellipsis
```

Comment:
u-ellipsis is defnined two times, one time for TextNode and one time for Text, I do not think it is necessary.